test/quantization/core/test_utils.py

```diff
@@ -192,30 +192,31 @@ def test_quantize_weight_clamping_per_channel(self):
         assert quantized_tensor.int_repr().max().item() == q8_max
         assert quantized_tensor.int_repr().min().item() == q8_min
 
-    def test_uint1_7_dtype(self):
+    def test_uint4_int4_dtype(self):
```

Comment:
what happened to the original test case?

Reply:
naming is a bit off, original test is testing uint4, and I added int4 in the PR